packages/offix-datastore/src/Model.ts

```diff
@@ -76,7 +62,7 @@ export class Model<T = unknown> {
     this.fields = config.fields;
     this.storage = storage;
     this.metadataName = metadataName;
-    // TODO set custom primary keys
+    // TODO set primary keys here or thru api
     this.storage.adapter.addStore({ name: this.storeName });
```

Comment:
I do not like that model executes and deals with its store. In my take model should be just stateless, immutable data container. Did not wanted to refactor this in this PR but that is must for the next phase IMHO